### DISCUSSION TITLE
`useTransition` needs to be replaced with `useNavigation`

### DISCUSSION THREAD

**Discussion by brandonpittman** [sergiodxa/remix-utils#178] [Q&A] [answered]
This is going to be a breaking change, isn't it?

**Comment by sergiodxa** [comment, marked as answer]
Yes, it's going to be a breaking change, it will need a new major once v2 is released

**Reply by brandonpittman** [reply]
May have to run patch-package till then. The console deprecation messages are already getting to me. 😊

**Reply by sergiodxa** [reply]
There’s a patch in Remix to remove them, it was not intended to be released yet.

**Reply by brandonpittman** [reply]
To remove the deprecation messages? Okay, I'll just sit on it then. Thanks.

**Comment by w00fz** [comment]
Hey @sergiodxa are you also thinking about renaming the util `useGlobalTransitionStates` to `useGlobalNavigationStates` ?

**Reply by sergiodxa** [reply]
Yes
